### PR TITLE
temple goodwill central city

### PR DESCRIPTION
**Pull request by GAV07**
db update for Temple University, Goodwill ORR, and Central City

### CODE REVIEW

File: db/vita_partners.yml [hunk 1]
```diff
@@ -484,3 +484,24 @@ vita_partners:
...
+  display_name: Temple University
+  source_parameters:
+  - tuvs
+  logo_path: 'partner-logos/temple.jpg'
+- name: Goodwill of the Olympics and Rainier Region
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360012588394'
+  display_name: Goodwill of the Olympics and Rainier Region
+  source_parameters:
+  - orrr
```
Comment (bengolder) [inline db/vita_partners.yml:499]: @GAV07 does `orrr` have the correct number of `r`s? it looks like the rest hints at just `orr`

File: db/vita_partners.yml [hunk 2]
```diff
@@ -484,3 +484,24 @@ vita_partners:
...
+  logo_path: 'partner-logos/goodwill_orr.png'
```
Comment (bengolder) [inline db/vita_partners.yml:500]: @GAV07 It looks like this image path does not match the file name

Comment (GAV07) [reply db/vita_partners.yml:500]: Yes, I forgot to add the extra 'r' to the link

File: db/vita_partners.yml (hunk 1 above)
Comment (GAV07) [reply db/vita_partners.yml:499]: Yes it is correct although I am not sure why they would have added an extra r instead of just two.

Comment (GAV07) [reply db/vita_partners.yml:499]: https://www.getyourrefund.org/?s=orrr (from [airtable](https://airtable.com/tbl5epnGDYQf8OAYZ/viwgXpuhMweqBKPIP/reck0uQ1GZjhbstX1?blocks=bipsOP1FkYFvEXxjd))

**Comment by bengolder** [comment]
Closing because changes were moved into this commit:
https://github.com/codeforamerica/vita-min/commit/4b7d72d92926473a702f0ac660ee721dcbf5889b

**Pull request closed without merging**